index.html

```diff
@@ -1,187 +1,212 @@
 
-      <div class="frameworks">
-        <p>Frameworks</p>
-        <ul class="list">
-          <li>Bootstrap</li>
-          <li>Ruby on rails</li>
-          <li>RSpec</li>
-          <li>Capibara</li>
-          <li>Selenium</li>
+        <ul>
+          <li class="blue-bg">HTML/CSS</li>
+          <li class="blue-bg">Ruby on Rails</li>
+          <li class="blue-bg">Javascript</li>
         </ul>
+        <div class="arrow-link">
+          <a class="link" href="#">See this Project</a>
+          <img class="arrow-icon" src="icons/Union.svg" alt="arrow" />
+        </div>
       </div>
+    </div>
+
+    <div class="about-container">
+      <img class="work1" src="works/work5.svg" alt="first project" />
 
-      <div class="skills">
-        <p>Skills</p>
-        <ul class="list">
-          <li>Codekit</li>
-          <li>Codepen</li>
-          <li>GitHub</li>
-          <li>GitLab</li>
-          <li>Terminal</li>
+      <div class="background-card">
+        <p class="project-name">Project name goes here</p>
+
+        <ul>
+          <li class="blue-bg">HTML/CSS</li>
+          <li class="blue-bg">Ruby on Rails</li>
+          <li class="blue-bg">Javascript</li>
         </ul>
+        <div class="arrow-link">
+          <a class="link" href="#">See this Project</a>
+          <img class="arrow-icon" src="icons/Union.svg" alt="arrow" />
+        </div>
       </div>
-    </section>
-  </body>
-</html>
+    </div>
+  </section>
+
+  <section>
+    <div class="about-section">
+      <h3>About</h3>
+      <p class="about-section-tag">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+        eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
+        minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+        aliquip ea eiusmod
+      </p>
+      <button class="dark-bg-cv" type="button">
+        Get my CV<img class="down-icon" src="icons/ic_down.svg" alt="arrow-down" />
+      </button>
+    </div>
+
+    <div class="languages">
+      <p>Languages</p>
+      <ul class="list">
+        <li>Javascript (ES5/ES6)</li>
+        <li>Ruby</li>
+        <li>HTML</li>
+        <li>CSS</li>
+      </ul>
+    </div>
+
+    <div class="frameworks">
+      <p>Frameworks</p>
+      <ul class="list">
+        <li>Bootstrap</li>
+        <li>Ruby on rails</li>
+        <li>RSpec</li>
+        <li>Capibara</li>
+        <li>Selenium</li>
+      </ul>
+    </div>
+
+    <div class="skills">
+      <p>Skills</p>
+      <ul class="list">
+        <li>Codekit</li>
+        <li>Codepen</li>
+        <li>GitHub</li>
+        <li>GitLab</li>
+        <li>Terminal</li>
+      </ul>
+    </div>
+  </section>
+
+  <section>
+    <div class="about-section get-started">
+      <h4>Get started</h4>
+      <p class="about-section-tag-start">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+        eiusmod tempor incididunt ut labore et dolore magna.
+      </p>
```

Comment:
- [ ] It would be great if you align this to be centered as shown in the Figma template to make your website identical to the Figma as much as poosible 🚀💯

- Your design
![image](https://user-images.githubusercontent.com/58553711/186174541-ad2644da-aa64-4ed2-801d-a782e02427a4.png)

- figma template 
![image](https://user-images.githubusercontent.com/58553711/186174748-6ca854af-074e-4099-a65a-db0d2fa80edb.png)